### PR TITLE
Feat/update endowed positions data 073124

### PR DESCRIPTION
**Pull request by wmui51**
# READY FOR REVIEW
- Update the schools and centers map

# Summary
- Update the schools and centers map and pages

# Review By (Date)
- @forrestglick can provide details on timing

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. check if the following pages work
- /endowed-positions/office-of-the-vice-provost-for-student-affairs
- /endowed-positions/office-of-the-vice-provost-for-undergraduate-education

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/endowed-positions/EndowedPositionsFooter.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const EndowedPositionsFooter = () => (
       <p>
         This site is a comprehensive directory of endowed positions at Stanford.
         It is updated six times per year, following meetings of Stanford&apos;s
-        Board of Trustees. Last updated April 29, 2024. For questions or
+        Board of Trustees. Last updated August 1, 2024. For questions or
```
Comment (yvonnetangsu) [inline src/components/endowed-positions/EndowedPositionsFooter.js:9]: Haven't we already merged this change into `dev`, @wmui51 ?

**Review by yvonnetangsu — APPROVED** [review]
Both pages work - GTG!

File: src/components/endowed-positions/EndowedPositionsFooter.js (hunk 1 above)
Comment (wmui51) [reply src/components/endowed-positions/EndowedPositionsFooter.js:9]: @yvonnetangsu i think so, i am not sure why these changes are still showing, let me try rebasing from dev.

Comment (wmui51) [reply src/components/endowed-positions/EndowedPositionsFooter.js]: i have rebased, and it looks like the previous commit is no longer there.

**Comment by forrestglick** [comment]
@yvonnetangsu Thanks for your help with this PR! I've reviewed the update, and all looks good to go. Please merge to Dev, and I will send for client review. She would like this to go live on Prod today, as it references our new President who is officially starting today.

**Comment by sherakama** [comment]
@forrestglick @wmui51 @yvonnetangsu code is on the way up to dev.

**Comment by forrestglick** [comment]
Thanks, everyone! I've sent the client the Dev link for review/approval. When we push this to Prod, the pending changes to the countdown timer will NOT be pushed as well, correct?

**Comment by yvonnetangsu** [comment]
@forrestglick Yes we will be careful not to include the countdown timer code in the release 👍🏼

**Pull request merged**